### DISCUSSION TITLE
How to Reduce Temperatures on ASUS ROG Zephyrus G16 (GU605MV) While Gaming: CPU Undervolting, G-Helper Adjustments, and Compatibility with Armoury Crate?

### DISCUSSION THREAD

**Discussion by TacoronteRiveroCristian** [seerge/g-helper#3045] [Q&A]
Hi everyone,

I have an ASUS ROG Zephyrus G16 (2024 model GU605MV), and I've noticed that during gaming sessions, particularly with Battlefield 1, the CPU temperatures range between 83-97 degrees Celsius, while the GPU stays around 80 degrees Celsius. I've updated all drivers and adjusted the in-game settings to optimize performance, but the CPU still runs very hot.

I've read about a technique called "undervolting" that might help reduce the CPU temperature by lowering the voltage it receives, which would prevent it from running at maximum power constantly. However, I'm not sure how to do this safely and effectively on this specific laptop model.

I've also seen that G-Helper has settings for power limits and disabling CPU Boost. Should I consider tweaking these settings as well? What impact could they have on temperatures and gaming performance?

Additionally, I’m curious if there’s any conflict between using G-Helper and Armoury Crate. Are there any known compatibility issues, or is it safe to use both?

I'd appreciate any advice or experiences you can share on how to perform CPU undervolting on my Zephyrus G16, whether it's advisable to adjust these other settings in G-Helper, and if there are any issues with running Armoury Crate alongside it.

Thanks in advance!

**Comment by seerge** [comment]
@CristianTacoronteRivero hello, those temperatures are absolutely fine (CPU below 96C and GPU below 86C). You can't undervolt Intel device, they are locked.

If you want you can limit CPU power to something like 30-40W or disable boost (which effectively also limits power consumption). Then it will run a bit cooler in games.

**Reply by TacoronteRiveroCristian** [reply]
Thanks for the clarification! By the way, aren't these temperatures bad for the other components near the CPU in the long run? Forgive my ignorance on these issues. And my last question is: don't G helper and armoury create interfere? Or is it better to uninstall armoury since I'm interested in using g helper?

Again, thanks for the answer!

**Reply by seerge** [reply]
@CristianTacoronteRivero they shouldn't. Your device is generally designed to run like that.

96C is a maximum temperature for CPU, when it hits it - it automatically starts dropping power consumption in order to cool down (it's called "throttling"). This is standard behavior for most modern laptops.

G-Helper and AC don't "interfere" directly, but they both adjust same settings. So the last one to set current mode - "wins" :) That's why it's advised to use something one in a long term, and uninstall [AC using official uninstaller](https://github.com/seerge/g-helper/wiki/Requirements#recommendations-optional)

**Reply by TacoronteRiveroCristian** [reply]
Thank you very much for your time!
